Jenkinsfile

```diff
@@ -388,8 +388,14 @@ pipeline {
         always {
           junit testResults: '**/target/surefire-reports/*.xml'
           container('maven') {
-            // clean up the redis namespace
-            sh "kubectl delete namespace ${NAMESPACE_REDIS} --ignore-not-found=true"
+            script {
```

Comment:
Not sure we need `script` here as we don't do script?

Reply:
yes, 
i have tested many solutions , that’s way we still have scripts  section.
i will  remove it